configure.ac

```diff
@@ -167,6 +167,13 @@ AC_ARG_ENABLE([zmq],
 
 AC_ARG_WITH([protoc-bindir],[AS_HELP_STRING([--with-protoc-bindir=BIN_DIR],[specify protoc bin path])], [protoc_bin_path=$withval], [])
 
+# Enable manpages
+AC_ARG_ENABLE(man,
+    [AS_HELP_STRING([--enable-man],
+                    [install man pages [default=yes]])],,
+    enable_man=yes)
```

Comment:
(I wonder if it should perhaps be disabled by default when building for Windows?)